### PR TITLE
Add note about support for LEAPP upgrade of FIPS systems

### PR DESCRIPTION
**Pull request by AkshayGadhaveRH**
Red Hat does not support LEAPP upgrades from RHEL 7 to RHEL 8 for systems that have FIPS enabled. As a workaround some users turn off FIPS to perform LEAPP upgrade and then turn it back on, however, this is not supported either and causes further issues. It is advisable to go with a fresh installation on a RHEL 8 system instead. Adding a note about supportability for LEAPP upgrade of FIPS enabled systems for clarification.

Bug link: https://bugzilla.redhat.com/show_bug.cgi?id=2159785

* [X] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only)
* [ ] Foreman 3.2/Katello 4.4 on EL7 & EL8
* [X] Foreman 3.1/Katello 4.3 on EL7 & EL8 (Satellite 6.11 EL7/8, orcharhino 6.1 on EL7, orcharhino 6.2 on EL7/8)
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc [hunk 1]
```diff
@@ -39,8 +39,11 @@ endif::[]
 ifdef::satellite[]
 [NOTE]
 ====
-{Project} supports DEFAULT and FIPS crypto-policies.
+- {Project} supports DEFAULT and FIPS crypto-policies.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc:42]: Please use `*` for unordered lists.

File: guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc [hunk 2]
```diff
@@ -39,8 +39,11 @@ endif::[]
 ifdef::satellite[]
 [NOTE]
 ====
-{Project} supports DEFAULT and FIPS crypto-policies.
+* {Project} supports DEFAULT and FIPS crypto-policies.
 The FUTURE crypto-policy is not supported for {Project} and {SmartProxy} installations.
+* In-place upgrade of {EL} systems in FIPS mode is not supported by {Team}.
+Turning FIPS off, upgrading from {EL} 7 to {EL} 8, and then turning FIPS on is not supported either.
+Instead, install {Project} {ProductVersion} on a freshly installed {EL} 8 system with FIPS mode enabled.
```
Comment (evgeni) [inline guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc:46]: Following this, you'd end up with a new, empty system. Which is probably not what people want :)

Can we link users up to the next section ("5. Migrating Satellite to a New Red Hat Enterprise Linux System") somehow? Something like "Follow the 'Migrating to new system' procedure with the new system in FIPS mode"?

**Comment by AkshayGadhaveRH** [comment]
@evgeni does this look good?

**Comment by evgeni** [comment]
The diff looks correct, yeah! Thanks!

File: guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc [hunk 3]
```diff
@@ -39,8 +39,12 @@ endif::[]
 ifdef::satellite[]
 [NOTE]
 ====
-{Project} supports DEFAULT and FIPS crypto-policies.
+* {Project} supports DEFAULT and FIPS crypto-policies.
 The FUTURE crypto-policy is not supported for {Project} and {SmartProxy} installations.
+* In-place upgrade of {EL} systems in FIPS mode is not supported by {Team}.
+Turning FIPS off, upgrading from {EL} 7 to {EL} 8, and then turning FIPS on is not supported either.
+Instead, migrate the {Project} {ProductVersion} to a freshly installed {EL} 8 system with FIPS mode enabled.
```
Comment (evgeni) [inline guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc:46]:
```suggestion
Instead, migrate {Project} {ProductVersion} to a freshly installed {EL} 8 system with FIPS mode enabled.
```

I'd drop the `the`, but it might be me?

**Review by evgeni — APPROVED** [review]

File: guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc (hunk 3 above)
Comment (AkshayGadhaveRH) [reply guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc:46]: Yeah, I am a little conflicted about it too. I felt like adding the article because we are now referring to the existing installation. @maximiliankolb, what do you think?

Comment (maximiliankolb) [reply guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc:46]: We generally don't use articles. Maybe reword to "migrate your "{ProjectServer}"? "Your" is often our alternative to "the" :D

Comment (AkshayGadhaveRH) [reply guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc]: @evgeni @maximiliankolb done! :D

**Comment by adamlazik1** [comment]
Merged to 3.1.
